library/src/main/java/de/agilecoders/wicket/webjars/collectors/AssetsMap.java

```diff
@@ -92,8 +92,15 @@ public String findRecentVersionFor(String path) {
     public Set<String> listAssets(final String folderPath) {
         final Collection<String> allAssets = getFullPathIndex().values();
         final Set<String> assets = new HashSet<String>();
-        final String prefix = settings.webjarsPath() + Helper.appendLeadingSlash(folderPath);
-
+        
+        final String prefix;
+        //Make sure the webjarpath does not already end with slash
+        if(!settings.webjarsPath().endsWith("/")){
```

Comment:
The default path is:
```java
this.webjarsPackage = "META-INF.resources.webjars";
this.webjarsPath = this.webjarsPackage.replace('.', '/');
```
I.e. there is no trailing slash.
Who adds the trailing slash in your case ? Some other library or your application ?